mappings/net/minecraft/client/render/entity/model/PlayerEntityModel.mapping

```diff
@@ -1,13 +1,13 @@
 CLASS net/minecraft/class_591 net/minecraft/client/render/entity/model/PlayerEntityModel
 	FIELD field_20787 parts Ljava/util/List;
-	FIELD field_3479 rightLegOverlay Lnet/minecraft/class_630;
+	FIELD field_3479 rightPantLeg Lnet/minecraft/class_630;
 	FIELD field_3480 thinArms Z
 	FIELD field_3481 ears Lnet/minecraft/class_630;
-	FIELD field_3482 leftLegOverlay Lnet/minecraft/class_630;
-	FIELD field_3483 bodyOverlay Lnet/minecraft/class_630;
-	FIELD field_3484 leftArmOverlay Lnet/minecraft/class_630;
+	FIELD field_3482 leftPantLeg Lnet/minecraft/class_630;
```

Comment:
```suggestion
	FIELD field_3482 leftPant Lnet/minecraft/class_630;
```

Reply:
How can you have a left pant, though? I didn't want to name it that because it implies each leg is wearing a full set of pants.

Reply:
Ah didn't realize "pant leg" is an actual term. My bad.

Reply:
np